Quran/AyahInfoPersistenceStorage.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  AyahInfoPersistenceStorage.swift
+//  Quran
+//
+//  Created by Ahmed El-Helw on 5/12/16.
+//  Copyright © 2016 Quran.com. All rights reserved.
+//
+
+import Foundation
+import SQLite
+
+struct AyahInfoPersistenceStorage {
+
+    struct Columns {
+        let id = Expression<Int>("glyph_id")
+        let page = Expression<Int>("page_number")
+        let sura = Expression<Int>("sura_number")
+        let ayah = Expression<Int>("ayah_number")
+        let position = Expression<Int>("position")
+        let minX = Expression<Int>("min_x")
+        let maxX = Expression<Int>("max_x")
+        let minY = Expression<Int>("min_y")
+        let maxY = Expression<Int>("max_y")
+    }
+
+    private let db: Connection
+    private let imageSize = "1920"
+    private let glyphsTable = Table("glyphs")
+    private let columns = Columns()
+
+    init() throws {
+        let file = String(format: "images_\(imageSize)/databases/ayahinfo_\(imageSize)")
+        guard let path = NSBundle.mainBundle().pathForResource(file, ofType: "db") else { fatalError() }
+        db = try Connection(path, readonly: true)
```

Comment:
Also, I would love to wrap the error comming from creating the connection into `PersistenceError`. So, we can show appropriate error message to the user in case of failure.